lib/cli.js

```diff
@@ -6,6 +6,7 @@ program
   .version(require('../package.json').version)
   .option('-o, --output <path>', 'Output folder for converted application. Defaults to ./.demeteorized.')
   .option('-r, --release <version>', 'The Meteor version. Defaults to latest installed.')
+  .option('-l, --arbitrary <\'{"xx":"xx"}\'> ', 'Add arbitrary JSON to package.json')
```

Comment:
I like the option to be `--json` or `-j`. Arbitrary is way too much to type out in a terminal